packages/server/src/server.ts

```diff
@@ -191,7 +207,7 @@ rpc.use('eth_blockNumber', async () => {
  * returns: Gas used - hex encoded integer
  */
 rpc.use('eth_estimateGas', async (params: any) => {
-  return logAndHandleResponse('eth_estimateGas', () => relay.eth().estimateGas(params?.[0], params?.[1]));
+  return logAndHandleResponse('eth_estimateGas', (requestId) => relay.eth().estimateGas(params?.[0], params?.[1],requestId));
```

Comment:
nit: here and instances below
```suggestion
  return logAndHandleResponse('eth_estimateGas', (requestId) => relay.eth().estimateGas(params?.[0], params?.[1], requestId));
```